cmd/iceberg/main.go

```diff
@@ -83,7 +83,13 @@ Options:
   --description TEXT 	specify a description for the namespace
   --location-uri TEXT  	specify a location URI for the namespace
   --schema JSON        	specify table schema in json (for create table use only)
-                       	Ex: [{"name":"id","type":"int","required":false,"doc":"unique id"}]`
+                       	Ex: [{"name":"id","type":"int","required":false,"doc":"unique id"}]
+  --properties TEXT 	specify table properties in key=value format (for create table use only)
+						Ex:"format-version=2,write.format.default=parquet"
+  --partition-spec TEXT specify partition spec as comma-separated field names(for create table use only)
+						Ex:"field1,field2"
+  --sort-order TEXT 	specify sort order as field:direction format(for create table use only)
+						Ex:"field1:asc,field2:desc"`
```

Comment:
do any of the existing iceberg CLIs (pyiceberg? any others?) have prior art for a create table via the CLI?

I'm still unsure why we need to support create-table through this CLI rather than just letting a user use code.

Reply:
I'm not sure if it's worthwhile to add it here, given how little code is needed to create a table. @kevinjqliu @Fokko what do you guys think?

Reply:
i think its helpful in certain scenarios. my only concern is with the proliferation of `(for create table use only)`. it was fine when we added only 1 parameter in #424. but now that there are many such parameters, maybe we can do sub-parameters specifically for the `create table` command.